README.md

```diff
@@ -40,7 +40,7 @@ with a different network, using whatever conventions a community chooses.
 Please visit [Technical Overview of Plenum](https://github.com/hyperledger/indy-plenum/blob/master/docs/main.md).
 
```

Comment:
I think it makes sense to put a reference to http://hyperledger-indy.readthedocs.io/projects/plenum here.

Reply:
Once we have actually built the readthedocs site, I will include readthedocs links for all of the documentation links that are relevant. For now, I think we can keep it as a github link.